adobecom/milo
# Add RUM click tracking to the sidekick library

We want more data on author behavior and track usage of the sidekick library in RUM for us to evaluate

Resolves: [MWPW-146673](https://jira.corp.adobe.com/browse/MWPW-146673)

## Test instructions
Alter the sidekick options (branch add-rum-data-sidekick-library--milo--adobecom, not stage)
<img width="1379" alt="Screenshot 2024-04-16 at 16 18 43" src="https://github.com/adobecom/milo/assets/39759830/8aed9786-ed3a-4e1b-9b03-dcaf642dd809">

Ensure the sidekick library still works
<img width="618" alt="image" src="https://github.com/adobecom/milo/assets/39759830/f40890b2-c091-4f90-931f-4c9a7f4a9e84">


**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/?martech=off
- After: https://add-rum-data-sidekick-library--milo--mokimo.hlx.page/?martech=off
